### DISCUSSION TITLE
Swipe animation

### DISCUSSION THREAD

**Discussion by festing** [olton/Metro-UI-CSS#1932] [General]
Seems like it should be easy....
How can I have element (ie: list item) move on X axis only, following the finger swipe to simulate a "swipe" on this?  I can't seem to figure out.
Any suggestions.  Seems like the "touch" swipe events should have some built in animation to solve this.

**Comment by MenesesEvandro** [comment]
Well... Not using MetroUI but it's very cool...

https://stackoverflow.com/questions/75586021/swipe-hand-icon-animation-with-css
